src/player.js

```diff
@@ -482,6 +484,35 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Checks for callbacks that should change the tracks, and call them on the
+   * respective track group (audio/text/video)
+   * @private
+   * @returns {void}
+   */
+  _maybeSetTracksLabels() {
+    const labelsCallbacks = this._config.customLabels;
+    if (labelsCallbacks) {
+      for (let callbackType in labelsCallbacks) {
```

Comment:
IMO there are too many usages of the word 'callbacks' (this is very confusing)
The above comment will help with this

Reply:
yep